README.md

```diff
@@ -0,0 +1,91 @@
+# Webpack BEM plugin
+
+[Webpack](https://github.com/webpack/webpack) plugin to build projects using [bem-react-core](https://github.com/bem/bem-react-core) and configured with [@bem.sdk-config](https://github.com/bem/bem-sdk/tree/master/packages/config)
+
+## Examples
+
+### Minimal example
+
+`bemrc.js`:
+
+```js
+module.exports = {
+    levels: [
+        { layer: 'common' }
+    ]
+};
+```
+
+in `webpack.config.js`:
+
+```js
+const WebpackBemPlugin = require('webpack-bem-plugin');
+
+// ...
+
+plugins: [
+    new WebpackBemPlugin()
+]
+```
+
+### Example with Sets
+
+`bemrc.js`:
+
+```js
+module.exports = {
+    levels: [
+        { layer: 'common' },
+        { layer: 'desktop' },
+        { layer: 'touch' }
+    ],
+    sets: {
+        desktop: 'common desktop',
+        touch: 'common touch'
+    }
+};
+```
+
+in `webpack.config.js`:
+
+```js
+const WebpackBemPlugin = require('webpack-bem-plugin');
+
+// ...
+
+output: {
+    path: '/path/to/project/build',
+    filename: '[setName].js'
+},
+plugins: [
+    new WebpackBemPlugin()
+]
+```
+
+## Plugin options
+
+```js
+new WebpackBemPlugin(options: object)
+```
+
+|Name|Type|Description|
+|----|----|-----------|
+|**`test`**|`{RegExp}`|Plugin processes `require` statements in matched files. Default is `/\.js$/`|
+|**`techs`**|`{String[]}`|List of techs in project to build. Typical values is `['js', 'css']`. Default is `['js']`|
+|**`techMap`**|`{Object}`|Maps techs to actual file extensions. Example: `{ js: 'react.js' }`|
+|**`libs`**|`{Object}`|Declares configurations for libraries defined in `libs` section of `bemrc`|
+
+## Run tests
+
+> npm test
+
+## Known issues
+
+* Small tests
+* Not picking changes in `bemrc` in watch-mode
```

Comment:
`Rereading `.bemrc` doesn't work in watch-mode`

Reply:
Yes, in plugin. May be its a problem. May be we should fix it in newer version of plugin so I noted it